workspaces/mcp-integrations/plugins/software-catalog-mcp-tool/dev/index.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createBackend } from '@backstage/backend-defaults';
```

Comment:
are the unit tests in the src dir using this??  .... based on how I read https://github.com/redhat-developer/rhdh-plugins/pull/1458/files#diff-bd65eba2b8c5ac5ab82ce15976dfc11e98246af61dbf58cb9b5be42731423214 I *think* the answer no, and this can be removed

or am I missing something @johnmcollier ?

Reply:
Not sure - but I removed it and didn't see any complaints when I ran the tests or accessed the MCP server manually.

Reply:
PR updated

Reply:
cool thanks ... fwiw that is what I typically do the time or two I've created a new plugin